Makefile

```diff
@@ -27,9 +27,9 @@ ISTIO_DOCKER_HUB ?= docker.io/istio
 export ISTIO_DOCKER_HUB
 ISTIO_GCS ?= istio-release/releases/$(VERSION)
 ISTIO_URL ?= https://storage.googleapis.com/$(ISTIO_GCS)
-ISTIO_CNI_HUB ?= gcr.io/istio-release
+ISTIO_CNI_HUB ?= gcr.io/istio-testing
```

Comment:
this does not contain the daily builds, so this will break things

Reply:
We publish daily builds to `gcr.io/istio-release`

Reply:
@sdake

Reply:
@howardjohn daily builids are published to istio-release only for cni, or for all repos?  (my question is around the proxy push specifically).